generated/nidcpower/nidcpower/errors.py

```diff
@@ -43,6 +43,20 @@ def __init__(self, code, description):
         super(DriverWarning, self).__init__('Warning {0} occurred.\n\n{1}'.format(code, description))
 
 
+class RpcError(Error):
```

Comment:
This should only be generated if gRPC is enabled for the API.

Reply:
Done